source/fundamentals/crud/read/limit.txt

```diff
@@ -0,0 +1,117 @@
+====================================
+Limit the Number of Returned Results
+====================================
+
+.. default-domain:: mongodb
+
+Use ``limit`` to cap the number of documents that can be returned from a
+read operation. The ``limit`` operation functions as a cap on the maximum number of
+documents that the operation can return, but it can return
+a smaller number of documents if there are not enough present
+to reach the limit. If ``limit`` is used with the
+``skip`` (TODO: link) method, the skip applies
+first and the limit only applies to the documents left over after
+the skip.
+
+Follow the instructions in the examples below to insert data into
+a collection and return only certain results from a query using a sort,
+a skip, and a limit. Perform the following operation to insert these documents
+representing books into a collection:
+
+.. code-block:: java
+
+    collection.insertMany(Arrays.asList(
+        new Document().append("_id", 1)
+            .append("title", "The Brothers Karamazov").append("length", 824)
+            .append("author", "Dostoyevksy"),
+        new Document().append("_id", 2)
+            .append("title", "Les Misérables").append("length", 1462).append("author", "Hugo"),
+        new Document().append("_id", 3)
+            .append("title", "Atlas Shrugged").append("length", 1088).append("author", "Rand"),
+        new Document().append("_id", 4)
+            .append("title", "Infinite Jest").append("length", 1104).append("author", "Wallace"),
+        new Document().append("_id", 5)
+            .append("title", "Cryptonomicon").append("length", 918).append("author", "Stephenson"),
+        new Document().append("_id", 6)
+            .append("title", "A Dance with Dragons").append("length", 1104)
+            .append("author", "Martin")
+    ));
+
+The following example queries the collection to return the top three
+longest books. It matches all the documents with the query, applies
+a ``sort`` on the ``length`` field to return books with longer lengths before
+books with shorter lengths, and applies a ``limit`` to return only ``3`` results:
+
+.. code-block:: java
+
+    // define a cursor that will return the first 3 sorted items
+    MongoCursor<Document> cursor = collection.find()
+        .sort(descending("length"))
```

Comment:
Vague method names like this can sometimes cause issues when users try to copy/paste the code, but can't quite get the imports right (particularly since method names like this tend to have overlaps with standard library methods). Consider following the pattern that @ccho-mongodb has used on other docs-java pages to demonstrate imports at the top of an example: https://github.com/mongodb/docs-java/blob/master/source/fundamentals/data-formats/documents.txt#L134